README.md

```diff
@@ -1,201 +1,35 @@
-[![Docker](https://img.shields.io/github/actions/workflow/status/HathorNetwork/hathor-core/docker.yml?branch=master&label=build&logo=docker)](https://hub.docker.com/repository/docker/hathornetwork/hathor-core)
 [![Codecov](https://img.shields.io/codecov/c/github/HathorNetwork/hathor-core?logo=codecov)](https://codecov.io/gh/hathornetwork/hathor-core)
-[![Discord](https://img.shields.io/discord/566500848570466316?logo=discord)](https://discord.com/invite/35mFEhk)
-[![License](https://img.shields.io/github/license/HathorNetwork/hathor-core)](./LICENSE.txt)
 
-## Running a full-node
+## Description
 
-**Disclaimer**
+**Hathor core** is the official client for operating a full node in Hathor Network.
 
-At the moment, our mainnet is running on a whitelist basis. This means only authorized nodes will be able to connect. For testing purposes, you can connect to the testnet (using the `--testnet` parameter). If you want to connect to the mainnet, you have to [use a peer-id](#using-a-peer-id) and send this id to a team member. You can get in touch with us through [our channels](https://hathor.network/community/), preferrably Discord.
+## Operation and usage
 
-### Using Docker
+To know how to operate and use Hathor core, see [Hathor full node at Hathor docs — official technical documentation of Hathor](https://docs.hathor.network/pathways/components/full-node).
 
-The easiest way to run a full-node is to use our Docker image. If you don't have Docker installed, check out [this
-link](https://docs.docker.com/install/). So, just run:
+## Support
 
-```
-docker run -ti -p 8080:8080 -p 8081:8081 hathornetwork/hathor-core run_node --cache --status 8080 --stratum 8081
-```
+If after consulting the documentation, you still need **help to operate and use Hathor core**, [send a message to the `#development` channel on Hathor Discord server for assistance from Hathor team and community members](https://discord.com/channels/566500848570466316/663785995082268713).
 
-The `--status 8080` will run our HTTP API on port 8080, while the `--stratum 8081` will run a stratum server on port
-8081. You can check your full-node status accessing `http://localhost:8080/v1a/status/`. Use `--help` for more
-parameters.
+If you observe an incorrect behavior while using Hathor core, we encourage you to [open an issue to report this failure](https://github.com/HathorNetwork/hathor-core/issues/new).
 
-For more information about our HTTP API, check out our [API Documentation](https://docs.hathor.network/).
+You can also [open an issue to request a new feature you wish to see](https://github.com/HathorNetwork/hathor-core/issues/new).
 
+Please do not open an issue to report a security breach or vulnerability. Instead, follow the guidelines described at [SECURITY](SECURITY.md) for safely reporting, fixing and disclosing security issues.
```

Comment:
These 3 lines seem all to be about contributing. And the third is exactly the same as the second line of "## Contributing".

What do you think of making the first line point to the Contributing section and removing the other 2 lines?

Reply:
Change made: 
I changed the sections "Support" and "Contributing" according to your suggestion.